packages/maker/dmg/src/MakerDMG.ts

```diff
@@ -17,15 +17,15 @@ export default class MakerDMG extends MakerBase<MakerDMGConfig> {
 
   async make({ dir, makeDir, appName, packageJSON, targetArch }: MakerOptions): Promise<string[]> {
     const { createDMG } = require('electron-installer-dmg');
-
-    const outPath = path.resolve(makeDir, `${this.config.name || appName}.dmg`);
+    const dmgName = `${this.config.name || appName}_${targetArch}`;
```

Comment:
Will this mean that all DMGs will have the arch suffix going forward?

1) I'd much prefer my DMGs not to have that extension.
2) I assume it would be a breaking change for some people. The way my update server is written, it would break if the DMG name suddenly changes.

IMO it should be configurable whether to have the arch in the file name (if it's even an option at all).

Reply:
ok, i will provide a option to enable or not